kunstakademie-munster.csl

```diff
@@ -13,57 +13,70 @@
     </author>
     <category citation-format="note"/>
     <summary>A German citation style used in art history and art sciences according to the standards of Academy of Fine Arts Münster. Deutschsprachiger Zitationsstil für Kunstwissenschaften nach Vorgaben der Kunstakademie Münster.</summary>
-    <updated>2020-06-29T16:22:30+00:00</updated>
+    <updated>2025-05-22T15:08:52+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <!-- ====== DEUTSCHE FORMEN ======================= -->
-  <locale xml:lang="de">
+<locale xml:lang="de">
     <terms>
       <term name="accessed">Zugriff am </term>
       <term name="editor" form="short">
         <single>(Hg.)</single>
         <multiple>(Hgg.)</multiple>
       </term>
+      <term name="director" form="short">(Regie)</term>
+	    <term name="page-range-delimiter">&#160;–&#160;</term>
       <term name="editortranslator" form="verb-short">hg. und übers. von</term>
       <term name="reviewed-author">Rezension zu</term>
-      <term name="et-al">et al.</term>
+      <term name="et-al">u.a.</term>
```

Comment:
You can remove this alltogether. The standard translation for et al. in the German locale is u.a.